src/validationinterface.h

```diff
@@ -6,10 +6,13 @@
 #ifndef BITCOIN_VALIDATIONINTERFACE_H
 #define BITCOIN_VALIDATIONINTERFACE_H
 
+#include <boost/signals2/signal.hpp>
+
```

Comment:
I believe this will slow down the build significantly. So tend to NACK here.

Reply:
Fixed